spacemeshos/smapp
# Fix: move back auto constant

it closes #1567 

How to test error cases:

**MacOS**:  run in terminal `tccutil reset AppleEvents` , this command will remove all automation programs, and on the app start you will see the permission modal on launch.
**Window**: for registry restriction, we can use [win 11 instructions](https://www.windowscentral.com/software-apps/windows-11/how-to-restrict-access-to-registry-on-windows-11) , for win 10 , from my understanding this scenario is not possible
**Linux**:  deb and AppImage , to catch the error need to clean up (in case for this PR version we already have auto launch script )`~/.config/autostart` folder and change permission for the folder `chmod a-w autostart` - for the autostart folder ( need to run this command in the upper folder ), return back `chmod +w autostart`

**AppImage**: in current release 1.2.5 , there is no config sync. And on start for new AppImage which they download the auto launch by default disabled, and only force enable based on the config value resolve this issue.
